docs/source/intro/tutorial.rst

```diff
@@ -570,6 +570,16 @@ Depending on which step the workflow is running, you should get something like t
 We can see that the ``MultiplyAddWorkChain`` is currently waiting for its *child process*, the ``ArithmeticAddCalculation``, to finish.
 Check the process list again for *all* processes (You should know how by now!).
 After about half a minute, all the processes should be in the ``Finished`` state.
+You can also see a nice overview of the steps the work chain with the ``verdi process status`` command:
+
+.. code-block:: console
+
+    $ verdi process status 19
+    MultiplyAddWorkChain<19> Finished [0] [3:result]
+        ├── multiply<20> Finished [0]
+        └── ArithmeticAddCalculation<22> Finished [0]
+
+This command can also be very useful when debugging a work chain.
```

Comment:
Why? Might be good to give a bit more detail

Reply:
How about (also considering your other comment [above](https://github.com/aiidateam/aiida-core/pull/4508#discussion_r511509657)):

> Between the final brackets next to the `MultiplyWorkChain` you can also see the current step in the outline (`[3:result]`). This information combined with the process state can be very useful when debugging a work chain.

Reply:
Yes, let's simply replace this sentence with

```
The bracket ``[3:result]`` indicates the current step in the outline of the :py:class:`~...MultiplyWorkChain` (step 3, with name `result`).
The ``process status`` is particularly useful for debugging complex WorkChains, since it helps pinpoint where a problem occurred.